src/open_inwoner/templates/pages/product/detail.html

```diff
@@ -3,139 +3,152 @@
 
 {% block sidebar_content %}
 {% anchor_menu anchors=anchors desktop=True %}
-{% if product.link %}
+{% if product.form_snippet %}
```

Comment:
Adding the form snippet here only adds it for desktop. So there will be no openforms integration for mobile.